pom.xml

```diff
@@ -22,29 +23,36 @@
         <skipTests>true</skipTests>
         <test.excludes>**/*Test.java.bogus</test.excludes>
 
-        <kfs.version>2019-08-08</kfs.version>
-        <rice.version>2.6.3</rice.version>
-        <cynergy.version>2.6.3</cynergy.version>
+        <kfs.version>2019-08-15</kfs.version>
+        <rice.version>2.7.0</rice.version>
+        <cynergy.version>2.7.0</cynergy.version>
         <cynergy.version.suffix>current</cynergy.version.suffix>
 
         <aws-java-sdk-dynamodb.version>1.11.18</aws-java-sdk-dynamodb.version>
         <backport-util-concurrent.version>2.2</backport-util-concurrent.version>
         <hibernate-validator-annotation-processor.version>5.4.1.Final</hibernate-validator-annotation-processor.version>
         <httpclient.version>4.5.5</httpclient.version>
         <jasperreports.version>2.0.4</jasperreports.version>
-        <javax.servlet-api.version>3.0.1</javax.servlet-api.version>
+        <javax.servlet-api.version>4.0.1</javax.servlet-api.version>
         <jersey.version>2.25.1</jersey.version>
         <jotm-core.version>2.1.10-kuali-1</jotm-core.version>
-        <jsp-api.version>2.2</jsp-api.version>
+        <jsp-api.version>2.2.1-b03</jsp-api.version>
```

Comment:
where does this version number come from? In base financials it is 2.3.1

Reply:
OK, I will fix